From 3b6315a2cf73156829e178a158d874998945a373 Mon Sep 17 00:00:00 2001
From: Karol Barski <karolx.barski@intel.com>
Date: Fri, 23 Sep 2016 07:30:56 -0500
Subject: [PATCH 171/213] drivers/pci: Increment MAX_MSI_IRQS in Axxia

Increase MAX_MSI_IRQS to 64.
Loading of igb driver tries to map 9 interrupts for each ethernet port
which results in requesting of more than 32 interrupts.

Signed-off-by: Karol Barski <karolx.barski@intel.com>
---
 drivers/pci/host/pcie-axxia.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/host/pcie-axxia.h b/drivers/pci/host/pcie-axxia.h
index a195c60..70b869a 100644
--- a/drivers/pci/host/pcie-axxia.h
+++ b/drivers/pci/host/pcie-axxia.h
@@ -19,7 +19,7 @@
  * it 32 as of now. Probably we will never need more than 32. If needed,
  * then increment it in multiple of 32.
  */
-#define MAX_MSI_IRQS			32
+#define MAX_MSI_IRQS			64
 #define MAX_MSI_CTRLS			(MAX_MSI_IRQS / 32)
 
 struct pcie_port {
-- 
1.9.1

